stores/src/main/java/com/issoft/training/stores/listCategories/Store.java

```diff
@@ -1,12 +1,17 @@
 package com.issoft.training.stores.listCategories;
 
+import com.issoft.training.domain.shop.Product;
 import com.issoft.training.domain.shop.categories.Category;
+import com.issoft.training.stores.comparaton.ComparatorByPrice;
+import com.issoft.training.stores.comparaton.ProductComparator;
+import com.issoft.training.stores.pars.DocPars;
 import com.issoft.training.stores.utils.RandomStorePopulator;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class Store {
-    private List<Category> categoryList;
+    private static List<Category> categoryList;
```

Comment:
you should not use static, before it is required and you fully understand what you are doing